trunk/NDHMS/Data_Rec/rt_include.inc

```diff
@@ -140,6 +144,24 @@
   REAL,  DIMENSION(50)     :: BOTWID,TOPWID,HLINK_INIT,CHAN_SS,CHMann !Channel parms from table
   REAL,  DIMENSION(50)     :: TOPWIDCC, NCC  !topwidth and mannings n of compund
 
+  ! VARIABLES FOR  RESERVOIRS
```

Comment:
These scalar lake variables are unnecessary. I just removed them and all tests passed. 
First red flag was that calars indicate that they would/should be local to some routine where they are used for intermediate calculations. 
Moreover, a major goal of this work is line: 
```
module module_levelpool_parameters

     use module_reservoir_base
    implicit none

     ! Extend/derive level pool parameters from the abstract base
    ! struct for reservoir properties.
    type, extends(reservoir_base_properties_struct) :: levelpool_parameters_interface
        real :: lake_area                ! area of lake (km^2)
        real :: weir_elevation           ! bottom of weir elevation (meters AMSL)
        real :: weir_coeffecient         ! weir coefficient
        real :: weir_length              ! weir length (meters)
        real :: orifice_elevation        ! orifice elevation (meters AMSL)
        real :: orifice_coefficient      ! orifice coefficient
        real :: orifice_area             ! orifice area (meters^2)
        real :: max_depth                ! max depth of reservoir before overtop (meters)
        integer :: lake_number           ! lake number

     contains

         procedure :: init => levelpool_parameters_init
        procedure :: destroy => levelpool_parameters_destroy

     end type levelpool_parameters_interface
```

Reply:
resolved by 663c47e